lib/schemas/get_manifest.json

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
This looks like a slightly tweeked copy of `post_manifest.json` -- was there no way to re-use the basic manifest schema w/o copying it over?